org.civicrm.reqangular/test/models/contact_test.js

```diff
@@ -267,5 +268,26 @@ define([
           .finally(done) && $rootScope.$digest();
       });
     });
+
+    describe('leaveManagees()', function () {
+      var contactID = "123";
+      var params = { key: 'value' };
+
+      afterEach(function() {
+        $rootScope.$digest()
+      });
+
+      it('calls leaveManagees function of contact API with same parameters', function () {
+        Contact.leaveManagees(contactID, params).then(function () {
+          expect(contactAPI.leaveManagees).toHaveBeenCalledWith(contactID, params);
+        });
+      });
+
+      it('returns contacts managed by the contact id', function () {
+        Contact.leaveManagees(contactID, params).then(function (data) {
+          expect(data).toEqual(contactAPIMock.mockedContacts().list);
```

Comment:
Yes it returns all the contacts, here I am testing if that return is working fine, because any way the filtering logic is part of backend, and in `contactAPIMock` whatever value is set to be returned, needs to be checked here if the same is returned by the model.